src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_approved.jelly

```diff
@@ -0,0 +1,204 @@
+            </j:if>
+            <j:choose>
+                <j:when test="${hashAndMetadataList.isEmpty()}">
+                    <p class="no-approved-scripts">
+                        <i>${%noApprovedScripts}</i>
+                    </p>
+                </j:when>
+                <j:otherwise>
+                    <table class="pane bigtable">
+                        <tr>
+                            <th width="1%"><!-- checkbox --></th>
+                            <th tooltip="${%approvedHash_tooltip}" class="info-cursor">${%approvedHash}</th>
+                            <th>${%approvedExpandCode}</th>
+                            <th>${%approvedLanguage}</th>
+                            <th title="${%approvedScriptLength_tooltip}" class="info-cursor">${%approvedScriptLength}</th>
+                            <th>${%approvedContextUser}</th>
+                            <th>${%approvedContextItem}</th>
+                            <th tooltip="${%usageCount_tooltip}" class="info-cursor">${%usageCount}</th>
+                            <th>${%lastTimeUsed}</th>
+                            <th title="${%wasPreapproved_header_tooltip}" class="info-cursor">
+                                <st:nbsp />
+                            </th>
+                            <th title="${%lastApprovalTime_tooltip}" class="info-cursor">${%lastApprovalTime}</th>
+                            <th>${%lastKnownApproverLogin}</th>
+                        </tr>
+                        <j:forEach var="hashAndMetadata" items="${hashAndMetadataList}">
+                            <j:set var="hash" value="${hashAndMetadata.hash}" />
+                            <j:set var="metadata" value="${hashAndMetadata.metadata}" />
+                            <j:set var="language" value="${metadata.language}" />
+                            <j:set var="hasScriptCode" value="${!metadata.isEmpty() and metadata.scriptLength gt 0}" />
+                            <j:set var="lastContext" value="${metadata.lastContext}" />
+                            <tr class="js-selectable-row">
+                                <td>
+                                    <input type="checkbox" value="" name="approved-hash"
+                                           id="check-approved-${hash}"
+                                           data-hash="${hash}"
+                                           class="js-checkbox-hash js-bypass-click" />
+                                </td>
+                                <td class="hash">
+                                    <label for="check-approved-${hash}" title="${hash}" class="js-bypass-click">${hash.substring(0, 8)}</label>
+                                </td>
+                                <j:choose>
+                                    <j:when test="${metadata.isEmpty()}">
+                                        <td colspan="10">
+                                            <i title="${%emptyMetadata_tooltip}">${%emptyMetadata}</i>
```

Comment:
Screenshots in the PR description would be nice!

Reply:
Actually you already have one :) Screenshots > "Full script already approved"

Reply:
Sorry, missed the collapsible section.